cms/static/js/spec/views/modals/move_xblock_modal_spec.js

```diff
@@ -67,6 +69,16 @@ define(['jquery', 'underscore', 'edx-ui-toolkit/js/utils/spec-helpers/ajax-helpe
             modal.show();
         };
 
+        isMoveEnabled = function(sourceIndex) {
+            var moveButton = modal.$el.find('.modal-actions .action-move')[sourceIndex];
```

Comment:
I think we don't need `sourceIndex` here and all the other places. There will be only one active modal at any time so we will only have a single `Move` button.